app/api/__init__.py

```diff
@@ -0,0 +1,79 @@
+""" These class defines a set of functions to used to validate information in the entire application """
+
+import re
+from passlib.hash import pbkdf2_sha256 as sha256
+from app.api.v2.models import cur
+
+class Tools():
+    # def __init__(self):
+    #     self.values = (int, float)
+
+    #A method to validate user information
+    @staticmethod
+    def validate_user_info(username, email, phone_number, password, confirm_password):
```

Comment:
You could refactor this method it's too nested adding the complexity of your code.